lib/server.gi

```diff
@@ -14,6 +14,30 @@ InstallSCSCPprocedure( "SCSCPStopTracing", SCSCPStopTracing,
     "To turn off tracing mode on the server", 
     0, 0 : force );     
 
+
+#############################################################################
+##
+##  SCSCP_TemporaryGlobalVarName( <prefix> ) returns a string that can be used
+##  as the name of a global variable that is not bound at the time when
+##  SCSCP_TemporaryGlobalVarName() is called. The optional argument prefix can
+##  specify a string with which the name of the global variable starts.
+##
+##  WARNING: this is not thread safe
+BindGlobal( "SCSCP_TemporaryGlobalVarName", function( prefix )
+    local nr,  gvar;
+
+    CheckGlobalName( prefix );
+    nr := 0;
+    gvar:= prefix;
+    while ISBOUND_GLOBAL( gvar ) do
+        nr := nr + 1;
+        gvar := Concatenation( prefix, String(nr) );
```

Comment:
@fingolfin hmm, what's the problem with using `RandomString(8)` as in the initial version?

Reply:
I don't understand, which initial version are you referring to? This is copied straight from GAP (well with some redundant checks removed)

Reply:
@fingolfin ah, I've misread the code. `RandomString(8)` is included in the prefix. So fine then.